__main__.py

```diff
@@ -58,15 +77,32 @@ def refresh_token():
 
 
 def run_optics_atef(config_file):
-    fn = ''
-    if os.path.isfile(config_file):
-        fn = config_file
-    elif os.path.isfile("./atefics/checkouts/" + config_file):
-        fn = "./atefics/checkouts/" + config_file
-    else:
-        print(f"{config_file} does not exist")
-        sys.exit(-1)
+    fn = FilePathUtil(config_file)
 
+    recipients = ['nrw@slac.stanford.edu', 'awallace@slac.stanford.edu']
+    msg = EmailMessage()    
+    msg['From'] = "lclsoptics@gmail.com"
+    msg['to'] = ", ".join(recipients)
+    msg['Content-Type'] = "text/plain"
+    content = ""
+
+    with open(client_id_path, 'r') as fd:
+        client_details = json.loads(fd.read())
+    current_hash = hash_config(config_file)
+    saved_hash = client_details['web']['rix_config_hash']
+
+    if current_hash != saved_hash:
```

Comment:
A bit of a more existential question: why are we doing this?  Is this config never going to update?  Are you afraid of someone accidentally modifying your config file?

On the technical side, if you're looking to compare config files I think you can compare them in python with `==`, but having the hash in the email is nice too.

Reply:
This is to ensure that the config doesnt change. That is, we want to guarantee we are checking the current values vs some unchanged values for radiation protection purposes. 
We could compare config files, but then we'd need to save an "original copy" of the file to compare it to.

Reply:
I guess I thought that having an "original copy" that's read only would be helpful later down the line, but this is fine too.